tools/ktlint

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+MAG="\033[95m"
+END="\033[0m"
+
+status() {
+    echo -e "$MAG$1$END"
+}
+
+ROOT=$(dirname $0)/..
+cd $ROOT
+
+status "Running ktlint on src_kt/..."
+# TODO(wkorman): Consider moving ktlint to a sigh task and (as we do
+# on Travis currently) only ktlint'ing modified files.
+find ./src_kt -name "*.kt" | xargs ktlint --android --relative --verbose .
```

Comment:
We should also run ktlint on `shells/android` and potentially particles/ & src/

Reply:
Yes, though this is all we do on Travis currently. Adding to the TODO.

Reply:
IIRC on travis we run on every kt file changed within the diff, regardless of directory

Reply:
Updated to do similar to travis but vs. `upstream/master`. Maybe we really want to diff vs whatever git's upstream branch is, but I figured just do this for now (and it logs that's what it's doing).